chsdi/models/vector/stopo.py

```diff
@@ -3412,3 +3412,18 @@ class GeologieFelslabore(Base, Vector):
     the_geom = Column(Geometry2D)
 
 register('ch.swisstopo.geologie-felslabore', GeologieFelslabore)
+
+
+class Gletscherausdehnung(Base, Vector):
+    __tablename__ = 'view_gletscherausdehnung_sgi_2016_glaciers'
```

Comment:
Maybe use a more generic name for future proving.

Reply:
agree with @b4l we should use a generic name here like ``view_gletscherausdehnung`` or ``view_gletscherausdehnung_tooltip``. it this layer will be updated one day, it will either replace the current data or create a time-enabled layer. We will have to find a generic / attribute / content based solution for the time-series then.

Reply:
@ltclm do you think is it enough if I change the name of the view for the moment. Or should I try to prepare something more for a potential update to make that as easy as possible in the future? For me the problem is a bit that we do not know exactly how the update will look.

Reply:
yes, thats enough for the moment, thank you!